src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -154,7 +154,10 @@ export class TechRecordSummaryComponent implements OnInit {
       this.setBrakesForces();
     }
 
-    this.vehicleTechRecordCalculated.noOfAxles = this.vehicleTechRecordCalculated.axles.length ?? 0;
+    if (this.vehicleTechRecord.vehicleType === (VehicleTypes.PSV || VehicleTypes.HGV || VehicleTypes.TRL)) {
+      this.vehicleTechRecordCalculated.noOfAxles = this.vehicleTechRecordCalculated.axles.length ?? 0;
+    }
```

Comment:
This doesn't work. For instance:

```js
enum Foobar {
    FOO = "foo",
    BAR = "bar"
}

const record = {
    bar: "bar"
}

if (record.bar === (Foobar.FOO || Foobar.BAR)) {
    console.log("this is not reached")
}
```

This is because under the hood, this is the comparison you're making:
```js
if ("bar" === ("foo" || "bar")) {
    console.log("you get a ts error when you do this")
}
```
which then evaluates to `"bar" === "foo"` which is obviously false. You even get a typescript error when you do that.

you need to expand it in this fashion:
```suggestion
    if (this.vehicleTechRecord.vehicleType === VehicleTypes.PSV || this.vehicleTechRecord.vehicleType === VehicleTypes.HGV || this.vehicleTechRecord.vehicleType === VehicleTypes.TRL)) {
      this.vehicleTechRecordCalculated.noOfAxles = this.vehicleTechRecordCalculated.axles.length ?? 0;
    }
```

Reply:
hmmm I hear you, I didn't think it would work either but something very similar is done above and:

`    if (this.vehicleTechRecord.vehicleType === (VehicleTypes.PSV || VehicleTypes.HGV || VehicleTypes.TRL)) {
      console.log('test');
      this.vehicleTechRecordCalculated.noOfAxles = this.vehicleTechRecordCalculated.axles.length ?? 0;
    }`

<img width="960" alt="Untitled" src="https://user-images.githubusercontent.com/114985934/210758206-2d74ae8b-c8c7-468f-9edb-d7032186c2ca.png">

Reply:
It also reaches the log on HGV, not just the first type